### DISCUSSION TITLE
idea-tips/plugin/javaer-top-5-free-plugins

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#27] [Announcements]
# idea-tips/plugin/javaer-top-5-free-plugins

今天，给大家推荐几个好用且免费的IntelliJ IDEA插件。如果你还没有用过，可以尝试一下，也许对你的日常工作会有一定的效率提升噢！ RESTFul-Tool 如果你是一个RESTful服务的开发者，那么这个一定要试一下。它是一套非常丰富的RESTful服务开发工具，对 Spring MVC 和 Spring Boot 支持也是非常友好，开发者可以直接通过插件找到自己编写的RESTful接口：

https://www.didispace.com/idea-tips/plugin/javaer-top-5-free-plugins.html

**Comment by xcodemap** [comment]
怎么能少了Record/Replay 的代码可视化分析引擎 xcodemap， https://xcodemap.tech/
